src/renderers/common/XRManager.js

```diff
@@ -904,7 +935,8 @@ class XRManager extends EventDispatcher {
 				const projectionlayerInit = {
 					colorFormat: gl.RGBA8,
 					depthFormat: glDepthFormat,
-					scaleFactor: this._framebufferScaleFactor
+					scaleFactor: this._framebufferScaleFactor,
+					clearOnAccess: false
```

Comment:
clearOnAccess gives a small benefit because it skips the step where browser explicitly clear the buffer upon framebuffer attach.